ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/bridge/validate/SequenceImpl.scala

```diff
@@ -172,37 +171,30 @@ private[validate] class SequenceImpl(
           inputContracts = txSubmission.inputContracts,
           completionInfo = completionInfo,
         )
-        _ <- deduplicateAndUpdateState(
+        recordTime = timeProvider.getCurrentTimestamp
```

Comment:
Source the record time and use it for:
* deduplication
* forwarding it in case of transaction accepted downstream

Reply:
Isn't that problematic for StaticTime implementation of the TimeProvider?

Reply:
Discussed offline. It is fine for deduplication to be record-time-based (which is based on the time mode of the ledger). This is in agreement with the KV committer behavior

Reply:
Clarified offline that record time is used consistently for deduplication also in the committer tests